server/raft.go

```diff
@@ -1933,9 +1933,13 @@ func (n *raft) applyCommit(index uint64) error {
 			// FIXME(dlc) - Check if this is us??
 			if _, ok := n.peers[oldPeer]; ok {
 				// We should decrease our cluster size since we are tracking this peer.
-				n.debug("Decreasing our clustersize: %d -> %d", n.csz, n.csz-1)
-				n.csz--
-				n.qn = n.csz/2 + 1
+				if n.csz > 1 {
+					n.debug("Decreasing our clustersize: %d -> %d", n.csz, n.csz-1)
```

Comment:
This means we could still go to 1, is that what we wanted here just to prevent going to 0, or did we want to also prevent going to 1?